src/main/cpp/Robot.cpp

```diff
@@ -319,20 +319,43 @@ void Robot::TeleopPeriodic() {
     frc::SmartDashboard::PutNumber("x", x);
     frc::SmartDashboard::PutNumber("y", y);
     frc::SmartDashboard::PutNumber("z", z);
+
+    photonlib::PhotonPipelineResult result = a_camera.GetLatestResult();
     if (!inDeadzone) {
         a_SwerveDrive.swerveUpdate(x, y, z, fieldOreo);
-    } else {
-        a_SwerveDrive.stop();
+    } 
+    else if(a_DriverXboxController.GetRightTriggerAxis() > 0.5) {
+         
+               
+      
+        if (result.HasTargets()) {
+            photonlib::PhotonTrackedTarget target = result.GetBestTarget();
+            double target_Yaw = target.GetYaw();
+            double goToYaw = a_Gyro.getAngleClamped() + target_Yaw;
+            frc::SmartDashboard::PutNumber("Yaw", goToYaw);
+            a_SwerveDrive.turnToAngle(goToYaw, true);
+        }
```

Comment:
@CMalohn16 / @VG-Fish : Something to consider for how this code might need to change is that we may not always want to snap to the best target. As an example, the speaker location has 2 April Tags. If we center on the wrong one, the bot may not be in the correct position to make the shot.